packages/io/svg-deserializer/index.js

```diff
@@ -161,23 +161,40 @@ const objectify = (options, group) => {
     shapes = shapes.map((shape) => {
       if ('transforms' in obj) {
         // NOTE: SVG specifications require that transforms are applied in the order given.
+        // But these are applied in the order as required by JSCAD
+        let tr = null
```

Comment:
Would be nice to see these a bit more readably named as 'transformRotation' or something similar